clang/include/clang/AST/Decl.h

```diff
@@ -5032,6 +5032,11 @@ class HLSLBufferDecl final : public NamedDecl, public DeclContext {
   SourceLocation KwLoc;
   /// IsCBuffer - Whether the buffer is a cbuffer (and not a tbuffer).
   bool IsCBuffer;
+  /// HasValidPackoffset - Whether the buffer has valid packoffset annotations
+  //                       on all declarations
+  bool HasPackoffset;
```

Comment:
I was looking around here because I was confused about why a validity check was calling `setHasPackoffset(IsValid)`.  The name in the comment - `HasValidPackoffset` - would not have confused me!

Reply:
Updated.